airbyte-integrations/connectors/source-mailerlite/metadata.yaml

```diff
@@ -1,13 +1,32 @@
 data:
+  allowedHosts:
+    hosts:
+      - connect.mailerlite.com
+  connectorBuildOptions:
+    # Please update to the latest version of the connector base image.
+    # https://hub.docker.com/r/airbyte/python-connector-base
+    # Please use the full address with sha256 hash to guarantee build reproducibility.
+    baseImage: docker.io/airbyte/python-connector-base:1.2.0@sha256:c22a9d97464b69d6ef01898edf3f8612dc11614f05a84984451dde195f337db9
   connectorSubtype: api
   connectorType: source
   definitionId: dc3b9003-2432-4e93-a7f4-4620b0f14674
-  dockerImageTag: 0.1.0
+  dockerImageTag: 1.0.0
   dockerRepository: airbyte/source-mailerlite
   githubIssueLabel: source-mailerlite
   icon: mailerlite.svg
   license: MIT
   name: MailerLite
+  releases:
+    breakingChanges:
+      1.0.0:
+        upgradeDeadline: "2024-06-30"
+        message:
+          "The verison migrates the MailerLite connector to the be compatable for connector builder.
+          Important:
+          The forms_popup stream schema from API has a breaking change to schema['properties']['settings']['properties']['schedule'],
```

Comment:
@btkcodedev is the breaking change related to the migration or has there been an API change?

Reply:
API field changes only